src/confluent_kafka/src/Producer.c

```diff
@@ -48,6 +48,7 @@
 struct Producer_msgstate {
 	Handle   *self;
 	PyObject *dr_cb;
+	PyObject *partitioner_cb;
```

Comment:
`partitioner_cb` shouldn't be a property of the message, it's something that should be set via the config.